### DISCUSSION TITLE
Auto change light/dark theme based on system settings

### DISCUSSION THREAD

**Discussion by tommyip** [grafana/grafana#44521] [Dashboards, Panels & Visualizations]
Both MacOS and Windows support system-wide dark mode and can be toggled on/off based on the current time. It would great if there is an option in Grafana to automatically change the theme according to system preference. This can be done using CSS media query [perfers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).

**Comment by mateuszdrab** [comment]
There needs to be an issue open for it. I sort of can't believe it's not implemented yet. 
Grafana is so eye sore in light more at dark, and the dark reader version isn't very readable. It needs an automatic switch just like many other tools have 😌

**Comment by gelicia** [comment]
Hello! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

This discussion was converted into an issue, which is available here: https://github.com/grafana/grafana/issues/73429
